src/Test/BlockServiceTestCase.php

```diff
@@ -24,7 +24,7 @@
  *
  * @author Sullivan Senechal <soullivaneuh@gmail.com>
  */
-abstract class AbstractBlockServiceTestCase extends TestCase
+abstract class BlockServiceTestCase extends TestCase
```

Comment:
This is a BC Break, this file is in `src`

Reply:
You should introduce a wrapper class and deprecate `AbstractBlockServiceTestCase`